sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientJavaDocCodeSnippets.java

```diff
@@ -74,90 +74,76 @@ public void detectLanguage() {
         String inputText = "Bonjour tout le monde";
         textAnalyticsAsyncClient.detectLanguage(inputText).subscribe(detectedLanguage ->
             System.out.printf("Detected language name: %s, ISO 6391 Name: %s, score: %.2f.%n",
-                detectedLanguage.getName(),
-                detectedLanguage.getIso6391Name(),
-                detectedLanguage.getScore()));
+                detectedLanguage.getName(), detectedLanguage.getIso6391Name(), detectedLanguage.getScore()));
```

Comment:
is there a reason why there are so many format changes in the PR?

Reply:
short line. Java use a tool to embeded code to readme, so maintaining a short line number is better for user experience.